### ISSUE TITLE
Docs.saltstack.com rendering develop branch docs for LTS branches

### ISSUE THREAD

**Issue by jhmcstanton** [saltstack/salt#53971]
### Description of Issue
The docs at docs.saltstack.com seem to be rendering the development branch docs for 2018.3.4. and 2019.2.0. See [salt.states.slack](https://docs.saltstack.com/en/develop/ref/states/all/salt.states.slack.html) for an example that includes webhook documentation (which is not available in 2018.3.4 or 2019.2.0 : [salt/states/slack.py](https://github.com/saltstack/salt/blob/v2019.2.0/salt/states/slack.py))

### Setup
n/a

### Steps to Reproduce Issue
See above links

### Versions Report
2018.3.4 and 2019.2.0

**Comment by dwoz** [comment]
@jhmcstanton Thanks for reporting this. Something definitely seems off here.

@bryceml Can you take a look at this?

**Comment by dwoz** [comment]
@jhmcstanton It looks like we're building the docs of the current head of each version branch, not the tag of what has been released. This has the benefit of allowing you to see what has been fixed since the last release but the downside of things getting a little out of sync before we make a relase. We're going to discuss whether or not this is desirable.

**Comment by jhmcstanton** [comment]
Thanks for checking this out. Honestly, needing to dig into the source code to find out what is actually available in our installation of saltstack adds a lot of extra time and headache when working on salt stories.

It would be great if the docs for the stable releases were in line with what is currently available.

**Comment by bryceml** [comment]
@dwoz I'm thinking maybe we should have point releases be available on docs in addition to the long running branches.

Right now we have 2018.3, latest, and develop building regularly based off the long running branch.

What if we had the point releases on there as well?  So 2018.3.4 could be on there, 2019.2.0 could be on there, etc.  I'm not sure if we would want latest to point to the latest released, or the latest in preparation to be released, probably the latest released and we could have branches like 2019.2.1 and neon being built regularly.

Basically what I'm proposing is we have:

* develop
* neon
* 2019.2.1
* 2019.2
* 2018.3
* 2017.7
* 2017.7.9

being built regularly

We can build 2019.2.0 once and make that be the default, or what latest points to.
We can build 2018.3.4 once
We can build 2017.7.8 once

Whenever we release, we can make sure that there is a build based off the tag and then stop building regularly.
